### PR TITLE
ESM + arcdown

### PR DESCRIPTION
**Pull request by tbeseda**
## done
remove all CJS
add WIP arc-render-md#esm
remove markdown cruft
tests are working

## broken
~~weird issue with arc-viewer wc styling~~
~~playground broken~~
~~☝️ these are related to a previous change to fingerprinted assets. fixing next~~
turns out this is an issue with brotli compression in new arc.functions

### CODE REVIEW

File: .github/workflows/link-checker.yaml [hunk 1]
```diff
@@ -1,61 +1,16 @@
...
-        run: npx arc hydrate
-        env:
-          CI: true
-
-      - name: check links
-        run: npm run link-checker
-        env:
-          CI: true
-
-      - name: Notify
```
Comment (macdonst) [inline .github/workflows/link-checker.yaml:53]: We don't want to notify discord of broken links?

File: src/http/any-catchall/index.js [hunk 2]
```diff
@@ -1,14 +1,11 @@
-// eslint-disable-next-line
```
Comment (macdonst) [inline src/http/any-catchall/index.js:1]: Can we just re-name these files to .mjs and then we can get rid of the package.json -> type: module files?

**Review by macdonst — APPROVED** [review]
Once we release the `arc-render-md` package I'd say we merge this too.

File: src/http/any-catchall/index.js (hunk 2 above)
Comment (tbeseda) [reply src/http/any-catchall/index.js:1]: That's a good point. I can't remember why that approach didn't work at first, but I will give it another shot and hopefully get rid of the stray package files

File: .github/workflows/link-checker.yaml (hunk 1 above)
Comment (tbeseda) [reply .github/workflows/link-checker.yaml:53]: I don't think so. We don't notify Discord when other tests break. The notifications has been used for deploys and publishing. the end game things.
If the link checker job fails, the PR's checks are marked as "not successful" just like tests or the woke job.

**Pull request merged**